scripts/console.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Should we add ShellCheck to the CI?

Reply:
This is an excellent point!
I already added the action.